flink-runtime/src/main/java/org/apache/flink/runtime/jobmanager/scheduler/ScheduledUnit.java

```diff
@@ -18,23 +18,21 @@
 
 package org.apache.flink.runtime.jobmanager.scheduler;
 
+import org.apache.flink.annotation.VisibleForTesting;
 import org.apache.flink.runtime.executiongraph.Execution;
 import org.apache.flink.runtime.instance.SlotSharingGroupId;
 import org.apache.flink.runtime.jobgraph.JobVertexID;
+import org.apache.flink.runtime.scheduler.strategy.ExecutionVertexID;
 import org.apache.flink.util.Preconditions;
 
 import javax.annotation.Nullable;
 
 /**
- * ScheduledUnit contains the information necessary to allocate a slot for the given
- * {@link JobVertexID}.
+ * ScheduledUnit contains the information necessary to allocate a slot for the given task.
  */
 public class ScheduledUnit {
 
-	@Nullable
-	private final Execution vertexExecution;
-
-	private final JobVertexID jobVertexId;
+	private final ExecutionVertexID executionVertexID;
```

Comment:
I would name this `executionVertexId` (compare with the member below: `slotSharingGroupId`)

Reply:
Ok. updated.